aws/cloudformation/standalone/gen_ai_curriculum/monitoring/dashboard/widgetCreators.ts

```diff
@@ -133,31 +133,29 @@ export function createBrowserLatencyComparisonGraph(
 export function createBrowserLatencyByModelGraph(
   models: {name: string; id: string}[],
   environment: string,
-  browser?: string
+  browser: string
 ): Widget {
   return {
     ...commonGraphProps,
     properties: {
       region: REGION,
       title: `Chat Request Latency by Model - ${browser || 'All Browsers'}`,
```

Comment:
Since `browser` is now required, is `|| 'All Browsers'` not needed? (same question for lines 183, 307 below).